src/NetAnalyzers/CSharp/Microsoft.CodeQuality.Analyzers/QualityGuidelines/CSharpRethrowToPreserveStackDetails.cs

```diff
@@ -62,6 +62,9 @@ private static void AnalyzeNode(SyntaxNodeAnalysisContext context)
                     case SyntaxKind.AnonymousMethodExpression:
```

Comment:
You could do so in a separate PR but it'd be nice to also have a test (if they are not already present) for action, func and local func.